PT_2011_IS_eusilc_cs.do

```diff
@@ -32,47 +32,47 @@ replace pt_dur = 3*4.3 	if country == "IS" & year == 2011 & pt_eli == 1
 /*	-> employed, self-employed: active for 6 months prior to birth
 			- employed for at least 25% of full time (10 hours/week for 40 hours/week full-time employment) 
 			- 80% earning

```

Comment:
Sorry I forgot to update the 80% earning to 80% earning below ... and 75% earning from ... like in 2012. I can make a new pull request if you want. Otherwise just see code from 2012. I did it correctly in 2010 though.